practice/validator/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Validator</title>
+  </head>
+  <body>
+        <form>
+          <p>
```

Comment:
Maybe not. I'm trying to avoid <div> tags, but those may be better than <p> tags in this instance. I do want the text and the input box to be grouped together though.